### ISSUE TITLE
Publish to packagist.com

### ISSUE THREAD

**Issue by paales** [magehost/performance-dashboard#3]
Would be nice if it were available on packagist.com.

**Comment by jeroenvermeulen** [comment]
Hi @paales,
Thanks for the tip. I just submitted it to Packagist.org:
https://packagist.org/packages/magehost/performance-dashboard
Is this what you mean?

**Comment by paales** [comment]
```
composer config repositories.magehost_performance-dashboard vcs https://github.com/magehost/performance-dashboard.git
composer require magehost/performance-dashboard:1.* --no-update
composer update magehost/performance-dashboard
```

Should be replaced with:
```
composer require magehost/performance-dashboard
```

- It is on packagist, so we dont need the composer config any more.
- for development run: `composer require magehost/performance-dashboard dev-master`
- It automatically locks to a mayor version, so 1.* isn't necessary anymore
- --no-update seems odd?
- composer update isn't required during install

👍

**Comment by jeroenvermeulen** [comment]
Fixed in commit c3e0bf9
